rest/blip_sync.go

```diff
@@ -296,6 +311,12 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, since db.SequenceID) {
 		}
 		return nil
 	})
+
+	// On forceClose, send notify to trigger immediate exit from change waiter
+	if forceClose && bh.user != nil {
+		bh.db.DatabaseContext.NotifyUser(bh.user.Name())
```

Comment:
I think `NotifyUser` is a really confusing function name.  This might be a good time to rename this to something more descriptive.  How about `DatabaeContext.StopChangeWaiter(username string)`?  That seems a lot more inline with the comment "trigger immediate exit from change waiter"

Reply:
The 'Notify' part of NotifyUser is important, because it's a reminder that this is just doing a broadcast over the tapNotifier - it's not doing any kind of synchronous stop of the user's changeWaiter.

I agree that the User part is bad - the current implementation doesn't even use the provided username, and should be refactored to get rid of that and just be NotifyTerminate.  It feels a bit risky to be tweaking for 2.0, though.  I could open a PR for dev/2.1 with that change, though.